docs/modules/ROOT/nav.adoc

```diff
@@ -22,8 +22,9 @@
 ** xref:cops_security.adoc[Security]
 ** xref:cops_style.adoc[Style]
 ** xref:v1_upgrade_notes.adoc[Upgrade Notes]
+* Plugin Documentation
```

Comment:
I think this section should be named simply "Plugins" or "Extensions". Probably the current "Extensions" page should be either split or renamed (to avoid duplication) a bit to something less generic.

Reply:
Thank you for your review. The section name has been changed to "Plugins". And the current "Extensions" page has been renamed to "Plugin Configuration", while keeping the filename as `extensions.adoc` to keep the URL.  

If a better way to split the content is found in the future, it can be updated. For now, the main aim is to move the "Plugin Migration Guide" page.